main.go

```diff
@@ -217,33 +229,56 @@ func logSavedReview(savedReview *SavedReview) {
 	log.Println("-------")
 }
 
-func saveReviewToFile(filePath, review string, reviewComments []*github.DraftReviewComment, action string) error {
+func saveReviewToFile(reviewFilePath, review string, reviewComments []*github.DraftReviewComment, action string) error {
+	// Save review content to .md file
+	mdFilePath := strings.Replace(reviewFilePath, ".json", ".md", 1)
+	err := os.WriteFile(mdFilePath, []byte(review), 0644)
+	if err != nil {
+		return fmt.Errorf("error saving review to .md file: %w", err)
+	}
+
+	// Save comments and action to .json file
+	jsonFilePath := reviewFilePath
 	savedReview := SavedReview{
-		Review:         review,
+		Review:         "", // Review content is stored in .md file
 		ReviewComments: reviewComments,
 		Action:         action,
 	}
-
 	data, err := json.MarshalIndent(savedReview, "", "  ")
 	if err != nil {
-		return err
+		return fmt.Errorf("error marshaling review comments and action to JSON: %w", err)
+	}
+	err = os.WriteFile(jsonFilePath, data, 0644)
+	if err != nil {
+		return fmt.Errorf("error saving review comments and action to .json file: %w", err)
 	}
 
-	return ioutil.WriteFile(filePath, data, 0644)
+	return nil
 }
 
-func loadReviewFromFile(filePath string) (*SavedReview, error) {
-	data, err := ioutil.ReadFile(filePath)
+func loadReviewFromFile(reviewFilePath string) (*SavedReview, error) {
```

Comment:
Ensure the path replacements handle edge cases correctly, especially if the file extensions change.